frontend/src/components/nav/navbar.jsx

```diff
@@ -87,27 +92,39 @@ const Logo = styled.img`
   height: 2rem; 
 `
 
-export default function Navbar({ stick, ele }) {
+export default function Navbar({sticky, ele}) {
+  const [out, setOut] = useState(false); 
   const dispatch = useDispatch();
   const loggedIn = useSelector((state) => state.session.isAuthenticated);
```

Comment:
I think you had this selector elsewhere, might be worth pulling out into a `selectors` file